ichspi.c

```diff
@@ -1704,7 +1704,7 @@ int ich_init_spi(void *spibar, enum ich_chipset ich_gen)
 	char *arg;
 	int ich_spi_rw_restricted = 0;
 	int desc_valid = 0;
-	struct ich_descriptors desc = {{ 0 }};
+	struct ich_descriptors desc = { 0 };
```

Comment:
Just learned something, didn't know that's valid in C (to leave out the braces).

Reply:
Urgh, it is, but then another warning pops up:

    error: suggest braces around initialization of subobject [Wmissing-braces]

I've switched this to just use an explicit memset instead.